controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -189,6 +189,23 @@ func (r *StorageClassRequestReconciler) reconcilePhases() (reconcile.Result, err
 		return reconcile.Result{}, err
 	}
 
+	profileName := r.StorageClassRequest.Spec.StorageProfile
+	if profileName == "" {
+		profileName = r.storageCluster.Spec.DefaultStorageProfile
```

Comment:
Q: (not related to this PR) will there be any chance that the default will change after installation?

Reply:
I agree, if the value changed it should only effect new CBPs